lib/bolt_server/transport_app.rb

```diff
@@ -45,9 +45,30 @@ def initialize(config)
 
       @file_cache = BoltServer::FileCache.new(@config).setup
 
+      # This is needed until the PAL is threadsafe.
+      @pal_mutex = Mutex.new
+      # These are only needed for the `/plans` endpoint.
+      require 'puppet'
```

Comment:
Why are these requires in the init method?

Reply:
Moved.